source/css/_common/components/post/post-meta.styl

```diff
@@ -16,6 +16,10 @@
     font-size: 14px;
     margin-top: 2px;
   }
+
+  time {
+    border-bottom: 1px solid $grey-dark;
```

Comment:
MB no need this styles?

Reply:
i was thinking MB a underline would helping user's to notice that there is more info about time.
not underline but MB something else would not confuse with href link.

Reply:
Yeah, need to think about it, but underline seems like it's hyperlink and really do some confuse. :smiley:

Reply:
MB we can replace solid to dotted?

Reply:
Try it.

Reply:
Or better dotted?
Try to play around with this: https://www.w3schools.com/cssref/pr_border-bottom_style.asp

Reply:
I use dashed.
[Done! and take a look!](https://jackey8616.github.io)